tests/execution.rs

```diff
@@ -3128,17 +3128,6 @@ fn test_load_elf_rodata_sbpfv1() {
     );
 }
 
-#[test]
```

Comment:
Is this test pointless now?

Reply:
Yep this is the default behaviour tested by `test_load_elf_rodata` now. And I added `test_load_elf_rodata_sbpfv1` that checks the same without virtual addresses.